pronto_quadruped_commons/include/pronto_quadruped_commons/forward_kinematics.h

```diff
@@ -45,18 +45,18 @@ class ForwardKinematics
     ForwardKinematics() = default;
     virtual ~ForwardKinematics() = default;
 
-    virtual Vector3d getFootPosLF(const JointState& q) = 0;
-    virtual Vector3d getFootPosRF(const JointState& q) = 0;
-    virtual Vector3d getFootPosLH(const JointState& q) = 0;
-    virtual Vector3d getFootPosRH(const JointState& q) = 0;
+    virtual Vector3d getFootPosLF(const JointState& q) { return getFootPos(q, LegID::LF); }
```

Comment:
I think we can simply remove those functions, they were just a proxy for the corresponding robcogen ones.